themes-default/slim/package.json

```diff
@@ -27,12 +27,13 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@vue/test-utils": "1.0.0-beta.21",
+    "@vue/test-utils": "1.0.0-beta.20",
     "ansi-colors": "2.0.5",
-    "ava": "1.0.0-beta.6",
+    "ava": "0.25.0",
     "axios": "0.18.0",
-    "babel-core": "6.26.3",
     "babel-plugin-istanbul": "4.1.6",
+    "babel-preset-env": "^1.7.0",
+    "babel-register": "^6.26.0",
```

Comment:
Please pin these two dependencies.
When you're adding new ones [use `--exact` or `-E` (on `yarn add`)](https://yarnpkg.com/lang/en/docs/cli/add/#toc-yarn-add-exact-e)

Reply:
Pinned